.travis.yml

```diff
@@ -12,59 +12,34 @@ matrix:
         apt:
           sources:
             - ubuntu-toolchain-r-test
-      env: PY=3
-    - os: linux
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-      env: PY=2
     - os: osx
       osx_image: xcode8
       compiler: clang
-      env: PY=3
 install:
     # Define the version of miniconda to download
     - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
         MINCONDA_OS=$MINCONDA_LINUX;
       elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
         MINCONDA_OS=$MINCONDA_OSX;
       fi
-    - if [[ "$PY" == "3" ]]; then
-          wget "http://repo.continuum.io/miniconda/Miniconda3-$MINCONDA_VERSION-$MINCONDA_OS.sh" -O miniconda.sh;
-      else
-          wget "http://repo.continuum.io/miniconda/Miniconda2-$MINCONDA_VERSION-$MINCONDA_OS.sh" -O miniconda.sh;
-      fi
+    - wget "http://repo.continuum.io/miniconda/Miniconda3-$MINCONDA_VERSION-$MINCONDA_OS.sh" -O miniconda.sh;
     - bash miniconda.sh -b -p $HOME/miniconda
     - export PATH="$HOME/miniconda/bin:$PATH"
     - hash -r
     - conda config --set always_yes yes --set changeps1 no
     - conda update -q conda
     # Host dependencies
-    - conda install xeus=0.19.1 nlohmann_json cppzmq = 4.3.0 -c conda-forge
+    - conda install xeus=0.19.1 nlohmann_json cppzmq=4.3.0 -c conda-forge
```

Comment:
You should upgrade to xeus 0.19.2 and install xtl 0.6.5

Reply:
yes